### PR TITLE
Fix script bug that prevents using logger

### PR DESCRIPTION
**Pull request by ChadNedzlek**

### CODE REVIEW

**Comment by ChadNedzlek** [comment]
Need to wait for my other PR to flow back into here to get the logger that does what I want.

File: eng/common/tools.ps1 [hunk 1]
```diff
@@ -358,7 +358,7 @@ function InitializeBuildTool() {
       ExitWithExitCode 1
     }
 
-    $buildTool = @{ Path = Join-Path $dotnetRoot "dotnet.exe"; Command = "msbuild" }
+    $buildTool = @{ Path = Join-Path $dotnetRoot "dotnet.exe"; Command = "msbuild"; Tool = "dotnet" }
```
Comment (tmat) [inline eng/common/tools.ps1]: Perhaps we can set `Framework = netcoreapp2.1` here?

File: eng/common/tools.ps1 [hunk 2]
```diff
@@ -367,7 +367,7 @@ function InitializeBuildTool() {
       ExitWithExitCode 1
     }
 
-    $buildTool = @{ Path = $msbuildPath; Command = "" }
+    $buildTool = @{ Path = $msbuildPath; Command = ""; Tool = "vs" }
```
Comment (tmat) [inline eng/common/tools.ps1]: Perhaps we can set `Framework = "net472"` here?

Comment (ChadNedzlek) [reply eng/common/tools.ps1]: Sure, that works. :-)

**Pull request merged**